### PR TITLE
Minimum number of deletions and insertions required to transform one string into another

### PR DESCRIPTION
**Pull request by AastikSharma05**
## Related Issue
- The problem is to tell the minimum number of insertion and deletion that need to be done in order to convert string 'a' into another string 'b'. ( #1247  )
- Added the code implementation of Minimum insertion and deletion to convert one string into another string using dp.
- Added the explanation of Minimum insertion and deletion to convert one string into another string Problem in the README.md file.

## Proposed Changes
- Added the code of Minimum insertion and deletion to convert one string into another string Problem using dp.
- Added README.md file of Minimum insertion and deletion to convert one string into another string Problem with time complexity.

## Additional Info
- This code uses dynamic programming.
- This problem is a variation of LCS problem.
- This is one of the commonly asked interview questions and the concept of this question helps us to solve many more problems.'
- This code solves an open issue #1247 
- I have tested these changes on VS code using multiple inputs and matched the output with the expected output.
- This change will not affect or create conflict with any other code.
- Time complexity: O(M*N)

## Checklist

- [x] ✅ My code follows the code style of this project.
- [x] 📝 My change requires a change to the README.
- [x] 🎀 I have updated the README accordingly.
- [x] 👀 I have read the [**CONTRIBUTING**](https://github.com/Lakhankumawat/LearnCPP/blob/main/Contributing.md) document.
- [x] ✨ I have added tests to cover my changes.
- [x] 🚩 All new and existing tests passed.
- [x] 🌟 ed the repo

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Output Screenshots
| Screenshot #1      | Screenshot #2  |
| ----------- | ----------- |
| Code Snippet  | Readme Snippet    |
| code | readme   |

### CODE REVIEW

**Comment by AastikSharma05** [comment]
@Mrjoy832  I am really sorry. Somehow the previous PR got closed if you don't mind can you please review this pr again .

**Comment by Mrjoy832** [comment]
image
do it

**Comment by AastikSharma05** [comment]
Done

**Comment by Mrjoy832** [comment]
LGTM🌟

File: D-DynamicProgramming/MinimumInsertionDeletion.cpp [hunk 1]
```diff
@@ -0,0 +1,51 @@
+#include <bits/stdc++.h>
+
+using namespace std;
+
+int lcs(string str1, string str2, int m, int n)
```
Comment (Lakhankumawat) [inline D-DynamicProgramming/MinimumInsertionDeletion.cpp:5]: Add Comments to your code

**Review by Lakhankumawat — CHANGES_REQUESTED** [review]

**Comment by AastikSharma05** [comment]
Added the comments in the CPP file @Lakhankumawat.

**Review by Lakhankumawat — APPROVED** [review]
🎉✨

**Comment by Lakhankumawat** [comment]
@all-contributors please add @AastikSharma05 code ,

**Pull request merged**